### ISSUE TITLE
Number of epochs required

### ISSUE THREAD

**Issue by nadimgh** [baidu-research/warp-ctc#32]
Hello,
Can I kindly ask for advice on the recommended number of epochs to be run to achieve non-blank predictions when using the CTC code? My network is always tending to predict a blank symbol, and I am thinking this is an effect of the number of run epochs (which is 30 for now).
Can someone give advice on the issue? Do you expect such a behavior to be an effect of the number of epochs, or the size of the training data?

Any help would be very much appreciated. Thank you !

**Comment by jaredcasper** [comment]
The number of epochs needed will be almost entirely dependent on the network architecture and data, so it is impossible to say how many epochs would be required just knowing that CTC is the loss function being used.  I will say, however, that if after 30 epochs you are only predicting the blank symbol something is definitely wrong outside of the CTC loss function.
